components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -60,6 +61,7 @@ type OpenshiftNotebookReconciler struct {
 	Namespace string
 	Scheme    *runtime.Scheme
 	Log       logr.Logger
+	Config    *rest.Config
```

Comment:
set this in `main.go`?

```
Config:    mgr.GetConfig(),
```

and also in `suite_test.go` for tests?

Reply:
Yes, we already have `Config: mgr.GetConfig(),` inside the [NotebookWebhook](https://github.com/opendatahub-io/kubeflow/blob/main/components/odh-notebook-controller/main.go#L142) struct initialization on main.go. As well as it exists on [suite_test.go](https://github.com/opendatahub-io/kubeflow/blob/main/components/odh-notebook-controller/controllers/suite_test.go#L189)